src/types/parameters.ts

```diff
@@ -3,6 +3,7 @@ import { Options } from './options';
 interface ParametersBase {
 	medium: Options['medium'];
 	warningLevel: Options['warningLevel'];
+	profile?: Options['profile'];
```

Comment:
Because `Options['profile']` is already optional, I don't think `?:` is necessary here. If that's correct (I haven't tested it), then for internal consistency with the other lines above it, let's remove the `?`:

```suggestion
	profile: Options['profile'];
```